### ISSUE TITLE
Missing ROS Doc

### ISSUE THREAD

**Issue by beetleskin** [locusrobotics/fuse#58]
As a user, it would be nice to have an overview about nodes, parameters, launch-files, API etc. Either on wiki.ros.org/fuse or docs.ros.org, or both.

**Comment by svwilliams** [comment]
Agreed. I am putting documentation together, but it is going slowly. Until then, what documentation is available is on the [github page](https://github.com/locusrobotics/fuse/blob/devel/README.md).

**Comment by DLu** [comment]
Close with https://docs.ros.org/en/api/fuse_doc/html/index.html?

**Comment by tim-fan** [comment]
It would be nice to have more documentation around how to configure the provided nodes via yaml. The "getting started" tutorial is nice in providing a specific example, but I'm left wondering what other options there are in terms of  motion models, sensor models, publishers etc, and how to go about configuring them. Also I'm not clear about which use-cases can be handled via config, and when I might need to start writing my own cpp to implement a constraint or model etc. 

I'm thinking in comparison about the nav2 docs, there are so many plugins etc available, and for each one you can generally count on there being corresponding documentation explaining how it works, and providing example configuration for it. It would be nice to have something similar for fuse.

**Comment by tim-fan** [comment]
I'd also note that most of the links under "further documentation" on the page https://docs.ros.org/en/api/fuse_doc/html/index.html are broken for me, not sure what's happening there.

**Comment by svwilliams** [comment]
> I'd also note that most of the links under "further documentation" on the page

Yes. I noticed the same thing recently. I'm looking into it.

**Comment by sloretz** [comment]
ROS Iron documentation seems to generate ok: http://docs.ros.org/en/iron/p/fuse_doc/, but curiously ROS Rolling does not http://docs.ros.org/en/iron/p/fuse_rolling/ 

I can't tell why from the doc job: https://build.ros2.org/job/Rdoc__fuse__ubuntu_jammy_amd64/14
